resources/lib/playerinfo.py

```diff
@@ -210,62 +217,24 @@ def update_total(self):
             pass
 
-    def push_position(self, position=0, total=100):
-        """Push player position to VRT NU resumepoints API"""
+    def push_position(self, position, total):
+        """Push player position to VRT NU resumepoints API and reload container"""
         # Not all content has an asset_id
         if not self.asset_id:
             return
 
+        # Set property to let wait_for_resumepoints function know that update resume is busy
+        set_property('resumepoints', 'busy')
+
         # Push resumepoint to VRT NU
         self.resumepoints.update(
             asset_id=self.asset_id,
             title=self.title,
             url=self.url,
             position=position,
             total=total,
-            whatson_id=self.whatson_id,
-            asynchronous=True
+            whatson_id=self.whatson_id
         )
 
-        # Kodi internal watch status is only updated when the play action is initiated from the GUI, so this doesn't work after quitting "Up Next"
-        if (not self.path.startswith('plugin://plugin.video.vrt.nu/play/upnext')
-                and not self.overrule_kodi_watchstatus(position, total)):
-            return
-
-        # Do not reload container when playing or not stopped
-        if self.isPlaying() or not self.quit.is_set():
-            return
-
-        container_reload()
-
-    @staticmethod
-    def overrule_kodi_watchstatus(position, total):
-        """Determine if we need to overrule the Kodi watch status"""
-
-        # Kodi uses different resumepoint margins than VRT NU, to obey to VRT NU resumepoint margins
-        # we sometimes need to overrule Kodi watch status.
-        # Use setting from advancedsettings.xml or default value
-        # https://github.com/xbmc/xbmc/blob/master/xbmc/settings/AdvancedSettings.cpp
-        # https://kodi.wiki/view/HOW-TO:Modify_automatic_watch_and_resume_points
-
-        ignoresecondsatstart = get_advanced_setting_int('video/ignoresecondsatstart', default=180)
-        ignorepercentatend = get_advanced_setting_int('video/ignorepercentatend', default=8)
-
-        # Convert percentage to seconds
-        ignoresecondsatend = round(total * (100 - ignorepercentatend) / 100.0)
-
-        if position <= max(SECONDS_MARGIN, ignoresecondsatstart):
-            # Check start margins
-            if SECONDS_MARGIN <= position <= ignoresecondsatstart:
-                return True
-            if ignoresecondsatstart <= position <= SECONDS_MARGIN:
-                return True
-
-        if position >= min(total - SECONDS_MARGIN, ignoresecondsatend):
-            # Check end margins
-            if total - SECONDS_MARGIN <= position <= ignoresecondsatend:
-                return True
-            if ignoresecondsatend <= position <= total - SECONDS_MARGIN:
-                return True
-
-        return False
+        # Set property to let wait_for_resumepoints function know that update resume is done
+        set_property('resumepoints', 'ready')
```

Comment:
So I don't understand why we wait for the online update, that is a liability IMO.
Only the local resumepoint updates should matter as that is freshly cached.